packages/admin/cms-admin/src/generator/future/generateForm/generateFormField.ts

```diff
@@ -115,6 +179,8 @@ export function generateFormField({
             fullWidth
             name="${nameWithPrefix}"
             label={${fieldLabel}}
+            ${config.startAdornment ? `startAdornment={<InputAdornment position="start">${startAdornment.adornmentString}</InputAdornment>}` : ""}
```

Comment:
does a text input really support startAdornment?

Reply:
Yes, it does, here's an example

<img width="1306" alt="Screenshot 2024-10-17 at 11 04 19" src="https://github.com/user-attachments/assets/f38a2697-41e3-4748-9d79-c1c79d7b903e">